hera_sim/visibilities/healvis_wrapper.py

```diff
@@ -0,0 +1,68 @@
+"""
+Wrapper around the healvis package for producing visibilities from
+healpix maps.
+"""
+
+from .simulators import VisibilitySimulator
+from healvis.simulator import setup_observatory_from_uvdata
+from healvis.sky_model import SkyModel
+from healvis.beam_model import AnalyticBeam
+from cached_property import cached_property
+import numpy as np
+import healpy
+
+
+class HealVis(VisibilitySimulator):
+    def __init__(self, fov=180, nprocesses=1, sky_ref_chan=0, **kwargs):
+        self.fov = fov
+        self._nprocs = nprocesses
+        self._sky_ref_chan = sky_ref_chan
+
+        # A bit of a hack here because healvis uses its own AnalyticBeam,
+        # and doesn't check if you are using pyuvsim's one. This should be fixed.
```

Comment:
It's probably worth discussing this too... I'm planning enable support for analytic beams in `pyuvdata.UVBeam` and have pyuvsim and healvis both use that instead (since both already carry `pyuvdata` as a dependency). The main reason we redefined analytic beams in `healvis` is because the analytic beams in pyuvsim are strictly E-field, and `healvis` only uses power beams.

Reply:
Is this enabled yet? Can I transition this code?